### PR TITLE
Update corporate-identifiers-add.md

### PR DESCRIPTION
**Pull request by msbemba**
Added Zero-touch and KME to the list of devices that gets corporate-owned status

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by PRMerger15** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/memdocs/issues/2386

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@lenewsad Please merge this PR. Thanks

**Review by lenewsad — APPROVED** [review]

**Comment by lenewsad** [comment]
#sign-off

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Comment by PRMerger9** [comment]
@tiburd : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: memdocs/intune/enrollment/corporate-identifiers-add.md [hunk 1]
```diff
@@ -37,6 +37,8 @@ As an Intune admin, you can identify devices as corporate-owned to refine manage
 At the time of enrollment, Intune automatically assigns corporate-owned status to devices that are:
 
 - Enrolled with a [device enrollment manager](device-enrollment-manager-enroll.md) account (all platforms)
+- Enrolled by using [Google Zero Touch](/android-dedicated-devices-fully-managed-enroll.md#enroll-by-using-google-zero-touch)
```
Comment (tiburd) [inline memdocs/intune/enrollment/corporate-identifiers-add.md:40]:
```suggestion
- Enrolled by using [Google Zero Touch](android-dedicated-devices-fully-managed-enroll.md#enroll-by-using-google-zero-touch)
```

**Comment by PRMerger3** [comment]
@tiburd : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**